corehq/apps/users/views/mobile/users.py

```diff
@@ -1396,3 +1398,33 @@ def post(self, request, *args, **kwargs):
             self.invitation.registered_date = datetime.utcnow()
             self.invitation.save()
         return self.get(request, *args, **kwargs)
+
+
+@method_decorator(TWO_STAGE_USER_PROVISIONING.required_decorator(), name='dispatch')
+class CommCareUserConfirmAccountView(TemplateView, DomainViewMixin):
+    template_name = "users/commcare_user_confirm_account.html"
+    urlname = "commcare_user_confirm_account"
+    strict_domain_fetching = True
```

Comment:
why is this necessary?

Reply:
Initially I copied it from the view I was working from, then figured out what it did, and then decided to keep it because:

1. if you change any password / settings on the domain (e.g. https://github.com/dimagi/commcare-hq/pull/26864#discussion_r394818458) I want to make sure those reflect immediately
2. I wouldn't expect this view to be accessed regularly so it seems fine to bust the cache

(I assume the only reason this flag exists is because we don't trust our domain cachebusting)